index.html

```diff
@@ -1,171 +1,435 @@
-<link rel="stylesheet" href="/stylesheets/asciinema-player.css">
-<link rel="stylesheet" href="/stylesheets/kb.css">
+  <meta charset="UTF-8">
+  <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="generator" content="Asciidoctor 1.5.2">
+  <title>Kill Bill userguide</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css"
+    integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://cdn.staticaly.com/gh/afeld/bootstrap-toc/v1.0.0/dist/bootstrap-toc.min.css" />
+  <link rel="stylesheet" href="stylesheets/asciinema-player.css" />
+
+  <!-- Bootstrap core CSS -->
+  <link href="https://getbootstrap.com/docs/4.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous" />
+
+  <!-- CSS extras -->
+  <link href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" rel="stylesheet" />
+  <link href="https://getbootstrap.com/docs/4.1/assets/css/docs.min.css" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400" />
+  <link rel="stylesheet" href="stylesheets/style.css" />
 </head>
-<body data-spy="scroll" data-target="#toc" class="book">
-<div class="container">
-  <div class="row" id="main-row">
-    <div id="header" class="col-sm-2 border-columns">
-      <div class="mx-auto text-center sticky-top">
-        <a href="/">
-          <img class="logo rounded mx-auto d-block sticky-top" src="https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/logo.png">
-        </a>
-      </div>
-      <nav id="toc" data-toggle="toc" class="toc sticky-top">
-        <ul class="nav navbar-nav">
-          <li><a class="nav-link active">Getting Started</a>
-            <ul class="nav navbar-nav">
-              <li><a class="nav-link" href="http://docs.killbill.io/latest/what_is_kill_bill.html">What is Kill Bill?</a></li>
-              <li><a class="nav-link" href="http://docs.killbill.io/latest/getting_started.html"><b>Quick start</b></a></li>
+
+<body>
+  <div class="container-fluid">
+    <div class="row flex-xl-nowrap">
+      <div class="col-12 col-md-3 col-xl-3 bd-sidebar">
+        <div class="mx-auto text-center sticky-top my-3">
+          <a href="/">
+            <img class="logo rounded mx-auto d-block sticky-top img-fluid"
+              src="https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/logo.png" />
+          </a>
+          <button class="btn btn-link bd-search-docs-toggle d-md-none p-0 ml-3 toggle-nav-btn" type="button"
```

Comment:
On my iPhone SE (1st generation), the button collides with the logo. Clicking on the button triggers the logo click instead of showing the menu.